File: app/src/main/java/br/com/victorlsn/mytaxi/beans/Coordinate.java
package br.com.victorlsn.mytaxi.beans;

/**
 * Created by victorlsn on 27/02/19.
 */

public class Coordinate {

    private double latitude;
    private double longitude;

    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    @Override
    public String toString() {
        return "Coordinate{" +
                "latitude=" + latitude +
                ", longitude=" + longitude +
                '}';
    }
}

File: app/src/main/java/br/com/victorlsn/mytaxi/beans/Coordinate.kt
package br.com.victorlsn.mytaxi.beans

/**
 * Created by victorlsn on 27/02/19.
 *
 */

class Coordinate {

    val latitude: Double = 0.toDouble()
    val longitude: Double = 0.toDouble()

    override fun toString(): String {
        return "Coordinate{" +
                "latitude=" + latitude +
                ", longitude=" + longitude +
                '}'
    }
}
